packages/core/src/components/Modal/Modal.stories.mdx

```diff
@@ -41,28 +40,30 @@ DummyComponent.displayName = 'DummyComponent';
 
 ## Shadow Effect
 
-Shadow will apear on the Header & Actions section if you scroll the content.
+The shadow effect apears on the Header & Actions section if you scroll through the content.
 
 ### Keyboard Support
 
-It also supports keyboard navigation when it is in focus
-
-| Key | Function        |
-| --- | --------------- |
-| Esc | Close the modal |
+The Modal component supports keyboard navigation when in focus. You can use the Esc key to close the modal.
```

Comment:
Could we please add the table back to maintain the consistency across the components.